docs/reference/gpu-arch-specs.rst

```diff
@@ -657,5 +657,54 @@ The following tables provide an overview of the hardware specifications for AMD
           - 256
           - 12.5
 
-For more information on the terms used here, see the :ref:`specific documents and guides <gpu-arch-documentation>`, the :doc:`conceptual overview of the HIP programming model<hip:understand/programming_model>`, or the :doc:`HIP reference guide<hip:reference/programming_model>`.
+Glossary
+########
 
+For more information on the terms used here, see the
+:ref:`specific documents and guides <gpu-arch-documentation>`, the
+:doc:`conceptual overview of the HIP programming model<hip:understand/programming_model>`,
+or the :doc:`HIP reference guide<hip:reference/programming_model>`.
```

Comment:
The last item not showing up as a hyperlink
![image](https://github.com/ROCm/ROCm/assets/164940351/bad4a0cc-2a9c-4345-ade1-c958989e5776)

The hyperlink names should also be the same as the destination titles.

Reply:
Fixed in #3075.